src/components/views/rooms/ReadReceiptGroup.tsx

```diff
@@ -44,8 +45,8 @@ interface Props {
 }
 
 // Design specified that we should show the three latest read receipts
-function determineAvatarPosition(index, count): [boolean, number] {
-    const firstVisible = Math.max(0, count - MAX_READ_AVATARS);
+function determineAvatarPosition(index, count, max): [boolean, number] {
```

Comment:
types, please

In tuples you can also give them labels e.g `[hidden: boolean, index: number]`

Reply:
I've replaced it with an object now, which is cleaner for readability anyway.